bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/PairVLQ_x53x53_tHtH_narrow/PairVLQ_x53x53_tHtH_narrow_RH_MX1100_MH400/PairVLQ_x53x53_tHtH_narrow_RH_MX1100_MH400_proc_card.dat

```diff
@@ -0,0 +1,7 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model X53_XSP_diagonalCKM_5FS_UFO 
+generate p p > x x~ , (x > t S+), (x~ > t~ S-)
+output PairVLQ_x53x53_tHtH_narrow_RH_MX1100_MH400 -nojpeg
```

Comment:
since this process using decay chain, and the mass is varying, the decay width needs to be set as "auto" to get a meaningful cross section. another question is the decay of "S+" and "S-" will be handled by pythia?

Reply:
Does this automatically set the decay width of the X5/3 ("x") or S+ or both? We have set the decay width to a nominal value in the UFO similarly to how it was done in previous years because we would like to simulate in the NWA. I think if we set it to auto we cannot assume NWA here anymore.